mobile/lib/models/metadata/file_magic.dart

```diff
@@ -96,6 +107,9 @@ class PubMagicMetadata {
       mvi: map[motionVideoIndexKey],
       noThumb: map[noThumbKey],
       mediaType: map[mediaTypeKey],
+      previewWidth: map[previewWidthKey],
+      previewHeight: map[previewHeightKey],
+      previewSize: map[previewSizeKey],
```

Comment:
Size is already available (including the metadata + video) in the fileData. Also, these previewHeight and Width should be stored along with the playlist as we might generate multiple previews for single video.